lib/json_api_controller/json_schema_validator.rb

```diff
@@ -7,7 +7,8 @@ module JsonSchemaValidator
 
     included do
       @action_params = Hash.new
-      (@actions & %i(update create)).each do |action|
+      schema_actions = %i(update create) | Array.wrap(@extra_schema_actions)
+      (@actions | schema_actions).each do |action|
```

Comment:
Changing this | to & (union back to intersection) causes `JsonApiController::NoValidatorForActionError` (from #params_for). Why should it be intersection? Don't I want to include the extra actions defined in @extra_schema_actions (in the ProPrefs controller) to be passed to #schema_class? Using & throws it away, hence the error.